diff --git a/node_modules/@web/test-runner-visual-regression/browser/commands.mjs b/node_modules/@web/test-runner-visual-regression/browser/commands.mjs
index a4b5d0d..a32e207 100644
--- a/node_modules/@web/test-runner-visual-regression/browser/commands.mjs
+++ b/node_modules/@web/test-runner-visual-regression/browser/commands.mjs
@@ -34,6 +34,10 @@ export async function visualDiff(element, name) {
       return;
     }
 
+    if (result.skip) {
+      return;
+    }
+
     if (typeof result.errorMessage === 'string') {
       throw new Error(result.errorMessage);
     }
diff --git a/node_modules/@web/test-runner-visual-regression/dist/visualRegressionPlugin.js b/node_modules/@web/test-runner-visual-regression/dist/visualRegressionPlugin.js
index bd4e733..33b55bc 100644
--- a/node_modules/@web/test-runner-visual-regression/dist/visualRegressionPlugin.js
+++ b/node_modules/@web/test-runner-visual-regression/dist/visualRegressionPlugin.js
@@ -23,6 +23,9 @@ function visualRegressionPlugin(options = {}) {
         async executeCommand({ command, session, payload }) {
             if (command === 'visual-diff') {
                 try {
+                    if (options.skip) {
+                        return { skip: true }
+                    }
                     if (!validatePayload(payload)) {
                         return;
                     }
